debian/clean

```diff
@@ -1,2 +1 @@
-debian/youtube-dl.1
-youtube_dl/*.pyc
+youtube_dl.egg-info/
```

Comment:
I think you might have removed the manually created man page from packaging some time ago, and *.pyc files are already taken care of by setup.py. Also, they would normally only be created for python2. You can check if everything is cleaned by deleting .gitignore, before building and cleaning.